.github/workflows/build.yml

```diff
@@ -130,14 +117,15 @@ jobs:
           AUTHENTICATOR_UUID=$(grep UUID -A1 -a $AUTHENTICATOR_PROFILE_PATH | grep -io "[-A-F0-9]\{36\}")
           cp $AUTHENTICATOR_PROFILE_PATH "$PROFILES_DIR_PATH/$AUTHENTICATOR_UUID.mobileprovision"
 
+      - name: Configure Ruby
+        uses: ruby/setup-ruby@943103cae7d3f1bb1e4951d5fcc7928b40e4b742 # v1.177.1
+        with:
+          bundler-cache: true
+
       - name: Install Mint, protobuf, xcbeautify, and yq
         run: |
           brew install mint swift-protobuf xcbeautify yq
-
-      - name: Install Mint packages
-        if: steps.mint-cache.outputs.cache-hit != 'true'
-        run: |
-          mint bootstrap
+          ./Scripts/bootstrap.sh
```

Comment:
🎨 In the PM repo we don't run the `bootstrap.sh` script as part of a CI run (for the build); we have the separate step for `mint bootstrap` and then the remainder of the bootstrap script is done in the `build.sh` script. This will therefore generate the Xcode project twice.

As well, when getting the build pipelines set up originally, I ran into issues where `mint bootstrap` wasn't properly picking up the cache (and therefore re-installing everything) but it looks like that's handled in the test pipeline somehow, so probably not an issue.

Reply:
Good catch. I didn't understand this. I will copy the iOS pattern.